ProjectTraning/QueueBooks/BookManagment.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProjectTraning.QueueBooks
+{
+    class BookManagment
+    {
+        public void StackExecute()
```

Comment:
Why do you add so many empty rows? Do you follow any logical structure in that case?